server/src/infra/infra.utils.ts

```diff
@@ -160,9 +160,19 @@ export function searchAssetBuilder(
     builder.andWhere(`${builder.alias}.ownerId IN (:...userIds)`, { userIds: options.userIds });
   }
 
-  const path = _.pick(options, ['encodedVideoPath', 'originalFileName', 'originalPath', 'resizePath', 'webpPath']);
+  const path = _.pick(options, ['encodedVideoPath', 'originalPath', 'resizePath', 'webpPath']);
   builder.andWhere(_.omitBy(path, _.isUndefined));
 
+  if (options.originalFileName) {
+    builder.andWhere(`${builder.alias}.originalFileName ILIKE :originalFileName`, {
+      originalFileName: `%${options.originalFileName}%`,
+    });
+
+    builder.orWhere(`${builder.alias}.originalPath ILIKE :originalFileName`, {
+      originalFileName: `%${options.originalFileName}%`,
+    });
+  }
+
```

Comment:
@mertalev, is there any concern about using `ILIKE` to search the content of these two columns?

Reply:
It'd be fast if you add a trigram index for this like for the geodata names. Also, the original path would cover both so no need to search on both.